### PR TITLE
RULEAPI-620 Update README

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -34,22 +31,67 @@ WARNING: Unlike the Search Page, the GitHub search across the PRs for unimplemen
...
+----
+
+Otherwise, if the rule has not been merged yet (i.e. the rule has not been implemented by any plugin), you can use the already existing PR corresponding to this rule. +
+
+Then,
+
+* Create a subdirectory with the name of the language you want to add. +
+It can be any of the following:
+include::supported_languages.adoc[]
+* Add two files in this subdirectory and write down the parts that need to be overwritten:
```
Comment (arseniy-sonar) [inline README.adoc:88]: Unlike Jira, you just write down the whole description in the rule.adoc (possibly including some parts using `include::` syntax).
Only metadata.json automatically inherits the parent metadata.json and overwrites parts of it.

File: README.adoc [hunk 2]
```diff
@@ -34,22 +31,67 @@ WARNING: Unlike the Search Page, the GitHub search across the PRs for unimplemen
...
+
+----
+Create rule S7028: All identifiers should be in CamelCase
+----
+
+Add the description of the PR to further increase its discoverability
+(GitHub PR search does not see the `rule.adoc`).
+To do that, click on the three dots (next to the smile) on the first comment (created by github-actions bot) and select "Edit".
+
+The rule must contain subdirectories corresponding to all the languages this rule is implemented for.
```
Comment (arseniy-sonar) [inline README.adoc:66]:
```suggestion
The rule must contain subdirectories corresponding to all the languages this rule is implemented for (by the time the PR is merged).
```

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]
Can you please, also, reformulate these sentences:
```
Go to the Search Page to find the rules that have already been merged in the master branch.

Go the the GitHub pull-request tab to find the rules that have not been merged yet. You can use the "Search in unimplemented" link on the Search Page to repeat your search in the PRs.
```

And fix the typo: "create a PR entitled" -> "create a PR named".

File: README.adoc [hunk 3]
```diff
@@ -167,16 +180,14 @@ The GitHub page that enables the search for rules.
...
 
-Here Jira RSPEC project becomes read-only, and the github repository becomes the source of truth.
-Once Jira RSPEC is frozen, the export script becomes unncessary and will be stopped.
-The https://sonarsource.github.io/rspec/#/[search page] taps directly into the GitHub repository and indexes the rules to
-replace the powerful search from the now-obsolete Jira database.
+== Help
+
+Tickets related to this RSPEC repository are in Jira, in the https://jira.sonarsource.com/projects/RULEAPI/issues/RULEAPI-324?filter=allopenissues[RULEAPI] project.
+
+You found a bug, something is bothering you or you have an idea to improve the project? First have a look at all the https://jira.sonarsource.com/projects/RULEAPI/issues/RULEAPI-324?filter=allopenissues[open tickets]. If you don't see anything related to your subject, please open a new ticket in the backlog, with `backlog` as fix version.
```
Comment (arseniy-sonar) [inline README.adoc:193]:
```suggestion
You found a bug, something is bothering you or you have an idea how to improve the project? First, have a look at all the https://jira.sonarsource.com/projects/RULEAPI/issues/RULEAPI-324?filter=allopenissues[open tickets]. If you don't see anything related to your subject, please open a new ticket in the backlog, with `backlog` as the fix version.
```

File: README.adoc [hunk 4]
```diff
@@ -8,48 +8,92 @@ This repository contains the specification of every static-analysis rule availab
...
+Otherwise, if the rule has not been merged yet (i.e. the rule has not been implemented by any plugin), you can use the already existing PR corresponding to this rule. +
+
+Then,
+
+* Create a subdirectory with the name of the language you want to add. +
+It can be any of the following:
+include::supported_languages.adoc[]
+* Add two files in this subdirectory:
+
+** `rule.adoc`: write down the whole description of the rule. If you want to include already existing parts (that are in the parent directory for example), use the `include::` syntax.
```
Comment (arseniy-sonar) [inline README.adoc:92]:
```suggestion
** `rule.adoc`: write down the whole description of the rule. If you want to include already existing parts (that are in the current or the parent directory), use the `include::` syntax.
```
rule-api does not support inclusion of files from any other directory, if I am not mistaken.

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 3 above)
Comment (arseniy-sonar) [reply README.adoc:193]: You are right, I think "of" is missing there as well. I meant to shift the meaning from

An idea to improve the project: "Hey, let's improve the project!"

to

An idea of how to improve the project: "Hey, let's add a script that would open a PR to add a language to an RSPEC automatically!"

Comment (amelie-renard-sonarsource) [reply README.adoc:193]: Oh, ok, makes sense

**Pull request merged**